src/stories/Library/search-result-page/search-result-page.scss

```diff
@@ -11,3 +11,41 @@
     gap: 16px;
   }
 }
+
+.facet-line {
+  &__item {
+    display: inline-block;
+    @extend %mr-8;
+    @extend %mt-8;
+
+    .dropdown__select {
```

Comment:
Please move these external classes into the dropdown styling as modifiers (dropdown__select and dropdown__arrows) 
That way they can be reused, live in the file corresponding to the component, and are not dependent on being inside an element with the facet-line class.